relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -211,20 +210,35 @@ fn scrub_resource_identifiers(mut string: &str) -> Option<String> {
     } else if let Some(pos) = string.find('#') {
         string = &string[..pos];
     }
+
+    // There can still me commas and semi-colons left in the URL.
+    if let Some(pos) = string.find(';') {
+        string = &string[..pos];
+    } else if let Some(pos) = string.find(',') {
+        string = &string[..pos];
+    }
+
     match RESOURCE_NORMALIZER_REGEX.replace_all(string, "$pre*$post") {
-        Cow::Owned(scrubbed) => Some(scrubbed),
+        Cow::Owned(scrubbed) => {
+            // We'll still want to normalize the domain.
+            if let Ok(url) = Url::parse(&scrubbed) {
+                let domain = normalize_domain(&url.host()?.to_string(), url.port())?;
+                return Some(format!("{}://{domain}{}", url.scheme(), url.path()));
+            }
+            Some(scrubbed)
```

Comment:
> we'll stop accepting resources without domain

IIUC, this will return the resource string as long as something was scrubbed, even if there's no domain. See tests below on lines 656 and 663.